src/org/ods/component/HelmDeploymentStrategy.groovy

```diff
@@ -95,8 +95,10 @@ class HelmDeploymentStrategy extends AbstractDeploymentStrategy {
         logger.info "Rolling out ${context.componentId} with HELM, selector: ${options.selector}"
         helmUpgrade(context.targetProject)
 
+        def deploymentKinds = DEPLOYMENT_KINDS << 'StatefulSet'
```

Comment:
@serverhorror, is this temporary, or do you plan to add 'StatefulSet' into the global DEPLOYMENT_KINDS?

Reply:
I suggest changing the _protected final_ field AbstractDeploymentStrategy.DEPLOYMENT_KINDS to _protected_ only in the concrete subclass and use a constant in class OpenShiftService instead of a plain String for 'StatefulSet'.